### PR TITLE
Add inversed by annotations on auto generate entities

### PR DESCRIPTION
**Pull request by luizkim**
Add inversed by annotations on auto generate entities.

Source:
http://w3facility.org/question/symfony2-doctrine2-generate-one-to-many-annotation-from-existing-database-by-doctrinemappingimport/#answer-17881139

### CODE REVIEW

**Comment by Ocramius** [comment]
@luizkim generating bi-directional associations is usually not a good idea...

**Comment by luizkim** [comment]
@Ocramius Maybe if we could pass it as parameters here:

$generator = new \Doctrine\ORM\Tools\EntityGenerator();        
**$generator->setBiDirectional(true);**
$generator->generate($metadata, $this->entityFolder);

Just google the number of people looking for how to do this and you will see that there are several .

**Comment by Ocramius** [comment]
> Just google the number of people looking for how to do this and you will see that there are several 

I am aware of this, we just don't want this to be the default.

Designing bi-directional by default is a bad idea.

**Comment by luizkim** [comment]
OK , i added via parameters :
```php
$driver = new \Doctrine\ORM\Mapping\Driver\DatabaseDriver(
                    $this->em->getConnection()->getSchemaManager()
            );
            $driver->setNamespace($this->namespace);
            $driver->setBiDirecionalEntities(true);
```

**Comment by Ocramius** [comment]
@luizkim let me rephrase before this goes further: we want to avoid people from doing this :-\

**Comment by Ocramius** [comment]
Closing as "won't fix" due to reasons explained above. The additional complexity is something we also don't really want to deal with for a feature that should not be used in first place.
